xcresultparser/main.swift

```diff
@@ -63,14 +69,20 @@ struct xcresultparser: ParsableCommand {
     
     private func outputDescription() throws {
         guard let resultParser = XCResultFormatter(
-                with: URL(fileURLWithPath: xcresultFile),
-                formatter: outputFormatter) else {
+            with: URL(fileURLWithPath: xcresultFile),
+            formatter: outputFormatter,
+            targets: targets
+        ) else {
             throw ParseError.argumentError
         }
         writeToStdOutLn(resultParser.documentPrefix(title: "XCResults"))
-        writeToStdOutLn(resultParser.summary)
```

Comment:
do we need the divider if notestresult == 0 ?
Maybe put all in one if-clause?
```
if notestresult == 0 {
    writeToStdOutLn(resultParser.summary)
    writeToStdOutLn(resultParser.divider)
    writeToStdOutLn(resultParser.testDetails)
}
if coverage == 1 {
    writeToStdOutLn(resultParser.coverageDetails)
}
```

Reply:
Done